src/root/file.d

```diff
@@ -73,7 +73,7 @@ struct File
             size_t size;
             stat_t buf;
             ssize_t numread;
-            char* name = this.name.toChars();
+            const(char)* name = this.name.toChars();
```

Comment:
I thought about that, and decided explicit was better when doing this conversion.